subgraphs/uniswap-forks/protocols/mm-finance/config/networks/cronos/cronos.ts

```diff
@@ -0,0 +1,114 @@
+import { Address, BigDecimal, BigInt } from "@graphprotocol/graph-ts";
+import { Factory } from "../../../../../generated/Factory/Factory";
+import {
+  BIGINT_ZERO,
+  FeeSwitch,
+  MINIMUM_LIQUIDITY_ONE_HUNDRED_THOUSAND,
+  MINIMUM_LIQUIDITY_ONE_THOUSAND,
+  Network,
+  PROTOCOL_SCHEMA_VERSION,
+  RewardIntervalType,
+} from "../../../../../src/common/constants";
+import { Configurations } from "../../../../../configurations/configurations/interface";
+import {
+  PROTOCOL_SUBGRAPH_VERSION,
+  PROTOCOL_METHODOLOGY_VERSION,
+  PROTOCOL_NAME,
+  PROTOCOL_SLUG,
+} from "../../../src/common/constants";
+
+export class MMFinanceCronosConfigurations implements Configurations {
+  getNetwork(): string {
+    return Network.CRONOS;
+  }
+  getSchemaVersion(): string {
+    return PROTOCOL_SCHEMA_VERSION;
+  }
+  getSubgraphVersion(): string {
+    return PROTOCOL_SUBGRAPH_VERSION;
+  }
+  getMethodologyVersion(): string {
+    return PROTOCOL_METHODOLOGY_VERSION;
+  }
+  getProtocolName(): string {
+    return PROTOCOL_NAME;
+  }
+  getProtocolSlug(): string {
+    return PROTOCOL_SLUG;
+  }
+  getFactoryAddress(): string {
+    return "0xd590cc180601aecd6eeadd9b7f2b7611519544f4";
+  }
+  getFactoryContract(): Factory {
+    return Factory.bind(
+      Address.fromString("0xd590cc180601aecd6eeadd9b7f2b7611519544f4")
+    );
+  }
+  getTradeFee(): BigDecimal {
+    return BigDecimal.fromString("0.17");
+  }
+  getProtocolFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.068"); // in their docs it's 0.07, but looking at the code is really 0.068
+  }
+  getLPFeeToOn(): BigDecimal {
+    return BigDecimal.fromString("0.102");
+  }
+  getProtocolFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0");
+  }
+  getLPFeeToOff(): BigDecimal {
+    return BigDecimal.fromString("0.17");
+  }
+  getFeeOnOff(): string {
+    return FeeSwitch.ON;
+  }
+  getRewardIntervalType(): string {
```

Comment:
The reward interval type should be per block should it not? It looks like you are calling a rewards per block function from the MasterChef contract (e.g. .try_meerkatPerBlock()). This should indicate the the rate that the MasterChef emits rewards is denominated by block intervals. This is important, because the quantity per interval and the interval allow to to calculate the projected rewards in the denominated token and the USD value respectively in the rewardsTokenEmissionsAmount and rewardTokenEmissionsUSD fields. You can see the rewards.ts file in src/common to see how this is done.

Reply:
Ooops, yeah it should be per block. 
Fixed 💪